cookbook/specs/solana.json

```diff
@@ -1196,6 +1196,26 @@
                                         "expected_value": "*"
                                     }
                                 ]
+                            },
+                            {
+                                "name": "tokens-owner-indexed",
```

Comment:
can you add a warning level for now?

Reply:
Fixed in [a79afde](https://github.com/lavanet/lava/pull/1676/commits/a79afde08d482172e4f1b1a89acd3c0f53e4157b)